### PR TITLE
Upgrade to Knot.x 1.5.0.

### PR DESCRIPTION
**Pull request by pmelkowski**
Upgrade the project to version 1.5.0

## Description
- changed the dependency to knotx-template-engine-handlebars
- changed the package of CustomHandlebarsHelper
- changed the name of the resource containing helper classes
- added the two helpers removed from Knot.x distribution

## Motivation and Context
Needed to use the new Knot.x template engine.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: src/main/java/io/knotx/handlebars/helpers/string/StringEqualsHelper.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * <pre>
+ *     {{#string_equals value "parameter"}}
+ *         equal
+ *     {{else}}
+ *         not equal
+ *     {{/string_equals}}
+ * </pre>
+ * If value is "parameter", the output will be "equal".
+ */
+public class StringEqualsHelper implements CustomHandlebarsHelper<Object> {
```
Comment (malaskowski) [inline src/main/java/io/knotx/handlebars/helpers/string/StringEqualsHelper.java:34]: It looks that newer version of jknack hbs have already `eq` helper:  https://github.com/jknack/handlebars.java/tree/v4.1.2#conditional-helpers

**Review by malaskowski — APPROVED** [review]
It looks that there is a lot of useful helpers in a newer version of `jknack/handlebars`: https://github.com/jknack/handlebars.java/tree/v4.1.2#built-in-helpers but I guess it can be addressed together with https://github.com/Knotx/knotx-template-engine/issues/13

**Comment by Skejven** [comment]
Please update README sections with new/changed extensions: https://github.com/Knotx/knotx-handlebars-extension#helpers

**Review by malaskowski — CHANGES_REQUESTED** [review]
Please update README sections with new/changed extensions: https://github.com/Knotx/knotx-handlebars-extension#helpers

**Review by malaskowski — APPROVED** [review]

**Pull request merged**